src/schema/objects/metadata.yaml

```diff
@@ -3543,7 +3543,7 @@ SourceDatasets:
   name: SourceDatasets
   display_name: Source Datasets
   description: |
-    Used to specify the locations and relevant attributes of all source datasets.
+    Used to specify the locations and relevant attributes of all source datasets (BIDS or not).
```

Comment:
This seems fine. Duplicating in the DOI and URL fields seems excessive.

Reply:
I think it is just a result of excessive (if not wrong to start with) specification in both of those. I do not see a need to make them "source dataset specific" and clearly should not relate to "all".